packages/manager/apps/web/client/app/hosting/cdn/flush/hosting-cdn-flush.controller.js

```diff
@@ -17,8 +18,9 @@ export default class HostingFlushCdnCtrl {
   }
 
   flushCdn() {
-    this.Hosting.flushCdn(this.$stateParams.productId)
-      .then(() => {
+    const isV1CDN = get(this.$scope.cdnProperties, 'version', '') === '2013v1';
+    const flushPromise = isV1CDN ? this.flushV1CDN() : this.flushSharedCDN();
+    flushPromise.then(() => {
```

Comment:
```suggestion
    return flushPromise.then(() => {
```

Reply:
initially there is no return, you think is required ?

Reply:
I think it's better and it is a good practice to do so. As it is asynchronous that way it can be chained and we can wait for the result to execute some other part of code if needed